package.json

```diff
@@ -519,6 +538,11 @@
           "when": "view == jobManager && viewItem == sqlJob",
           "group": "inline"
         },
+        {
+          "command": "vscode-db2i.jobManager.editSelfCodes",
+          "when": "view == jobManager && viewItem == sqlJob",
+          "group": "inline"
```

Comment:
I don't think this should be inline right now, since we already had 4 icons. I think the group should be in the SELF group. Perhaps:

```
"group": "self@2"
```